include/glow/Importer/ProtobufLoader.h

```diff
@@ -104,6 +104,10 @@ class ProtobufLoader {
   /// \returns the tensor that was registered under the name \p name.
   Tensor *getTensorByName(llvm::StringRef name);
 
+  /// \returns the tensor that was registered under the name \p name, or null if
+  /// no such tensor exists.
+  Tensor *getTensorByNameOrNull(llvm::StringRef name);
```

Comment:
Before having this method, we normally did checks like `if (tensors_.count(name))`. Does it make sense to switch them to use the new method?

Reply:
I'm just going to use tensors_.count() to perform the check.  I don't think there's too much need to add more API, since that's a member anyways.